fiscal_epos_print_fiscalcode/static/src/js/fiscal_epos_print_fiscalcode.js

```diff
@@ -0,0 +1,27 @@
+odoo.define("fiscal_epos_print_fiscalcode.models", function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var _t = core._t;
+    var fiscal_epos_print_models = require("fiscal_epos_print.epson_epos_print");
+
+    fiscal_epos_print_models.eposDriver.include({
+        printFiscalReceiptHeader: function(receipt){
+            var fiscalcode_message = "";
+            var current_client = this.sender.pos.get_client();
+            var fiscalcode = current_client && current_client.fiscalcode;
+            if (fiscalcode && current_client.epos_print_fiscalcode_receipt) {
+                fiscalcode = fiscalcode.toUpperCase();
+                var fiscalcode_message = _t("F.C.: ") + fiscalcode + "\n";
```

Comment:
Non serve un `\n` anche prima di `"F.C`? Altrimenti se `receipt.header` contiene già qualcosa non gli viene appiccicato `F.C.:`?

Reply:
Grazie della review!
Non so come siano fatti di solito gli header e ho dato per scontato che in fondo avessero un 'a capo'.
Ho fatto la modifica che hai suggerito